src/platforms/hosted/cli.c

```diff
@@ -421,6 +422,9 @@ void cl_init(bmda_cli_options_s *opt, int argc, char **argv)
 			if (optarg)
 				opt->opt_gpio_map = optarg;
 #endif
+		case 'k':
+			opt->opt_cmsisdap_allow_fallback = true;
+			break;
```

Comment:
Need to `break` the gpiod case as well, else implicit fallthrough (build with libgpiod-dev)